### DISCUSSION TITLE
Is SSLv3 supported in openssl3.0?

### DISCUSSION THREAD

**Discussion by Shaluw** [openssl/openssl#25445] [Q&A]
Is SSLv3 supported in openssl 3.0?
I have set the security level to 0. But still it does n't work.
Even I see this option is removed from s_client as well.
As per the documentation, sslv3 should work after setting the security level to 0.
![image](https://github.com/user-attachments/assets/dea0f56a-4aa5-47dd-8267-328bbcb76767)

Can someone please confirm if its supported or not

**Comment by mattcaswell** [comment]
Support for SSLv3 is not compiled in by default. There is a build time option to enable it (enable-ssl3).

**Reply by FdaSilvaYY** [reply]
> Support for SSLv3 is not compiled in by default. There is a build time option to enable it (enable-ssl3).

and `enable-weak-ssl-ciphers` to enable some old ciphers related to SSL3, no ?

**Reply by Shaluw** [reply]
Thanks for the quick answer. I will try compiling by enabling sslv3.
